DevCycle.SDK.Server.Local/Api/DVCLocalClient.cs

```diff
@@ -245,7 +245,7 @@ private VariableType_PB TypeEnumToVariableTypeProtobuf(TypeEnum type)
             catch (Exception e)
             {
                 logger.LogError("Unexpected exception getting variable: {Exception}", e.Message);
-                return null;
+                return Common.Model.Local.Variable<T>.InitializeFromVariable(null, key, defaultValue);
```

Comment:
Do you need the full package path for the variable? You could probably do this with a static import

Reply:
Oh I see the issue.  It is because in that context C# can't tell if i'm talking about the DVCLocalClient.Variable<T>() method or the type so it won't compile unless I provide the path

Reply:
Ahh. Good ol overlapping scopes.